### PR TITLE
Add management routes for composite views.

### LINKED ISSUE #2102: CompositeView: Implement the mgmt routes

### PR DESCRIPTION
**Pull request by wwajerowicz**
Fixes #2102

### CODE REVIEW

File: delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewRejection.scala [hunk 1]
```diff
@@ -248,4 +259,43 @@ object CompositeViewRejection {
...
+  implicit val compositeViewHttpResponseFields: HttpResponseFields[CompositeViewRejection] =
+    HttpResponseFields {
+      case RevisionNotFound(_, _)            => StatusCodes.NotFound
+      case TagNotFound(_)                    => StatusCodes.NotFound
+      case ViewNotFound(_, _)                => StatusCodes.NotFound
+      case ViewAlreadyExists(_, _)           => StatusCodes.Conflict
+      case IncorrectRev(_, _)                => StatusCodes.Conflict
+      case WrappedProjectRejection(rej)      => rej.status
+      case WrappedOrganizationRejection(rej) => rej.status
+      case UnexpectedInitialState(_, _)      => StatusCodes.InternalServerError
```
Comment (umbreak) [inline delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewRejection.scala:298]: CompositeViewEvaluationError should be handled as InternalServerError

**Review by umbreak — CHANGES_REQUESTED** [review]

File: delta/plugins/composite-views/src/test/resources/routes/responses/view.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      ],
+      "includeDeprecated": false,
+      "@type": "ProjectSource",
+      "_uuid": "{{uuid}}"
+    }
+  ],
+  "projections": [
+    {
+      "@id": "http://example.com/blazegraph-projection",
+      "query": "SELECT * WHERE {?s ?p ?p}",
```
Comment (bogdanromanx) [inline delta/plugins/composite-views/src/test/resources/routes/responses/view.json:55]: select queries should fail, shouldn't they?

**Review by bogdanromanx — CHANGES_REQUESTED** [review]
It's missing the module binding.

File: delta/plugins/composite-views/src/test/resources/routes/responses/view.json (hunk 2 above)
Comment (wwajerowicz) [reply delta/plugins/composite-views/src/test/resources/routes/responses/view.json:55]: yes, this query is not correct, I just wanted to have something short to keep the example responses/requests short. I can change it for another query.

Comment (umbreak) [reply delta/plugins/composite-views/src/test/resources/routes/responses/view.json:55]: I've created an issue for the query bit: https://github.com/BlueBrain/nexus/issues/2220

**Review by bogdanromanx — APPROVED** [review]

**Review by umbreak — DISMISSED** [review]

**Review by bogdanromanx — APPROVED** [review]

**Review by umbreak — APPROVED** [review]

**Pull request merged**